docs/configuration.md

```diff
@@ -378,6 +378,18 @@ various parts of the Emacs integration.
 In particular, `lsp-haskell` provides customization options for the `haskell-language-server`-specific parts,
 such as the path to the server executable.
 
+#### Vanilla Emacs
```

Comment:
Is this specific for Vanilla Emacs? I think it's applicable Emacs in general

Reply:
I saw documentation for doom and spacemacs, so I assumed that would be a better method for non-vanilla users, and us vanilla users would benefit from setting the config directly.

Reply:
Other sections explain the installation method, which actually corresponds to the explanation above this subsection.

Reply:
I think I see what you're saying now. How about this tweak? (https://github.com/haskell/haskell-language-server/blob/c6065259e294e397b285dc6bed8c7b38d3a9aefd/docs/configuration.md#emacs)

Reply:
Looks good to me :)